overwolf/src/logic/featureFlags.ts

```diff
@@ -0,0 +1,8 @@
+const forbiddenFeaturesKey = 'DANGER_ENABLE_FORBIDDEN_FEATURES';
+
+const enableForbiddenFeatures = localStorage.getItem(forbiddenFeaturesKey) === true.toString();
```

Comment:
If necessary, this can also be changed to a constant `false` in the repository, and `true` during development.